devcontainer.json

```
@@ -0,0 +1,30 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+	"name": "Test_container",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:0-20",
```

Node v16 is the most current we can use atm
```suggestion
	"image": "mcr.microsoft.com/devcontainers/javascript-node:0-16",
```